infrastructure/food_oasis_app_stack/templates/task-definition.json

```diff
@@ -0,0 +1,42 @@
+[
```

Comment:
This application is a little different from the other and it may need a volume mounted which would be reflected in this definition file

Reply:
right, lets discuss this on ops call